web/src/hotkey.js

```diff
@@ -213,27 +213,111 @@ const character_mappings = {
     "@": {name: "compose_reply_with_mention", message_view_only: true},
 };
 
+function is_printable_ascii(key) {
+    return key.length === 1 && key >= " " && key <= "~";
```

Comment:
It'd be helpful to have a comment documenting what set of characters this is meant to cover. It's not at all obvious to me that this corresponds exactly to 'printable ascii" characters...

Reply:
Added comment
```
    // ASCII printable characters (character code 32-126) -> " " to "~".
    // It includes letters, digits, punctuation marks, and a few
    // miscellaneous symbols.
```